assets/tests/components/inputModals/deletePresetModal.test.tsx

```diff
@@ -6,7 +6,13 @@ import { initialState, closeInputModal } from "../../../src/state"
 import { StateDispatchProvider } from "../../../src/contexts/stateDispatchContext"
 
 describe("DeletePresetModal", () => {
-  test("can enter a name and save", () => {
+  test("can confirm deletion", () => {
+    const originalFS = window.FS
+    window.FS = { event: jest.fn(), identify: jest.fn() }
+    afterEach(() => {
+      window.FS = originalFS
+    })
```

Comment:
*recommendation*: this mock setup happens often enough that it might be worth refactoring out into a helper method, but it certainly doesn't have to be.